plugin.rb

```diff
@@ -630,8 +631,7 @@ def to_hash
 
     def self.find(id, opts = {})
       if DataExplorer.pstore_get("q:#{id}").nil? && id < 0
-        json = JSON.parse(File.read(File.expand_path("../config/default_queries.json", __FILE__)))
-        hash = json["queries"][id.to_s].with_indifferent_access
+        hash = Queries.default[id.to_s]
```

Comment:
@SamSaffron, simply indexing by id here and below. Should I rework this to find by value and not key?

Reply:
At the end of the file you can rework your query hash to index by id, or allow id to be text which is kind of nice